dotCMS/src/main/java/com/dotcms/rest/api/v1/authentication/ApiTokenResource.java

```diff
@@ -272,50 +271,53 @@ public final Response revokeApiToken(
     @NoCache
     @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
     @Consumes(MediaType.APPLICATION_JSON)
```

Comment:
**Question regarding issues #30268 and #30142**:  
The `@Consumes` annotation was introduced in the previous PR for this ticket. Since the endpoints for `revoke` and `delete` don't receive a body, is it correct to define this annotation here?

Reply:
Good catch, that's correct. `@Consumes` isn't harming anything, but it's also not doing anything in those endpoints, and can be safely removed. I'll add another commit to snip those.